__tests__/app.test.js

```diff
@@ -53,14 +53,50 @@ describe('/api',()=>{
                         description: expect.any(String),
                         queries: expect.any(Array),
                         exampleResponse: expect.any(Object)
+                    }))
+                }
+            }
+        })
+    })
+})
 
+describe('api/articles/:article_id',()=>{
+    test('GET:200 responds with an object asscoiated with the article id',()=>{
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body:{articles}})=>{
```

Comment:
seeing as we're only ever expecting one article back (the one specified by the id) we should just send this back as an object rather than sending back the array of one object, this means the client will not need to process the array to extract out that one object